PiwikTracker/URLSessionDispatcher.swift

```diff
@@ -94,9 +94,15 @@ final class URLSessionDispatcher: Dispatcher {
 }
 
 fileprivate extension Event {
+    
+    private func cvarParameterValue() -> String {
+        var cvars: Array<String> = customVariables.enumerated().map { "\"\($0.offset + 1)\":[\"\($0.element.name)\",\"\($0.element.value)\"]" }
+        return "{\(cvars.joined(separator: ","))}"
```

Comment:
I think this code is a bit to complex to parse. Do you think it is possible to optimize it?